esx_service/utils/kvESX.py

```diff
@@ -175,24 +185,28 @@ def load(volpath):
     except IOError:
         return None
 
-    kvDict = json.load(fh)
-
+    kv_str = fh.read()
     fh.close()
+    try:
+       kv_dict = json.loads(kv_str)
```

Comment:
why this try/except is needed ? Does not upstairs need to check for return value anyways, so it could haqve handles exceptions instead?  I think we need to catch exception only when a decision is needed, and up to this moment we can simply pass it up (which is default behavior)